lib/tasks/cms_seeds.rake

```diff
@@ -2,35 +2,37 @@
 
 namespace :comfy do
   namespace :cms_seeds do
-    desc "Import CMS Seed data into database (from: folder name, to: site identifier)"
+    desc "Import CMS Seed data into database (from: folder name, to: site identifier, klasses: class name[s])"
 
-    task :import, %i[from to] => [:environment] do |_t, args|
+    task :import, %i[from to klasses] => [:environment] do |_t, args|
       from  = args[:from]
       to    = args[:to] || from
+      klasses = args[:klasses].nil? ? nil : args[:klasses].split
```

Comment:
`classes = args[:classes]&.split`. This will split if it's not nil. Otherwise it's nil.